test/Confluent.Kafka.IntegrationTests/Tests/AssignPastEnd.cs

```diff
@@ -23,12 +23,12 @@
 
 namespace Confluent.Kafka.IntegrationTests
 {
-    /// <summary>
-    ///     Test functionality of Consumer.Consume when assigned to offest
-    ///     higher than the offset of the last message on a partition.
-    /// </summary>
     public static partial class Tests
     {
+        /// <summary>
+        ///     Test functionality of Consumer.Consume when assigned to offest
+        ///     higher than the offset of the last message on a partition.
+        /// </summary>
```

Comment:
this example doesn't use kafka offset management at all - it just calls Assign on a particular partition with an offset based on the last produced message, so the auto.offset.reset property should be irrelevant.. 

However, now you've got me looking at this... the test should check the produced message offset it not invalid. Also I noticed the Offset property in Message is still of type long not Offset and I'll change that too.